torch/csrc/autograd/variable.h

```diff
@@ -436,6 +434,10 @@ struct TORCH_API Variable::Impl : public at::TensorImpl {
 
   int64_t storage_offset() const override;
 
+  void set_version_counter(const c10::VariableVersion& version_counter) noexcept override;
```

Comment:
curious: can you not put override on the implementation below?

Reply:
Just tried and yes we can :)